runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -2,14 +2,22 @@
 
 import com.codahale.metrics.MetricRegistry;
 
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.util.concurrent.ThreadFactoryBuilder;
+import io.netty.channel.ChannelOption;
+import io.netty.channel.EventLoopGroup;
+import java.lang.Thread.UncaughtExceptionHandler;
 import java.time.Duration;
+import java.util.AbstractMap;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.AbstractMap'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)